src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -460,6 +472,19 @@ private String getGitBranchName(Supplier<String> gitCommand) {
             return gitCommand.get();
         }
 
+        private static Optional<String> getLocalBranch(String remoteBranch) {
+            // This finds the longest matching remote name. This is because, for example, a local git clone could have
+            // two remotes named `origin` and `origin/two`. In this scenario, we would want a remote branch of
+            // `origin/two/main` to match to the `origin/two` remote, not to `remote`
```

Comment:
Do you possibly mean:

```suggestion
            // `origin/two/main` to match to the `origin/two` remote, not to `origin`
```

Reply:
Yes 🙃